cluster/addons/registry/README.md

```diff
@@ -189,6 +189,12 @@ metadata:
     kubernetes.io/cluster-service: "true"
     version: v0.4
 spec:
+  selector:
```

Comment:
These are defaulted from the template metadata labels. Please omit.

Reply:
thanks ,   registry  has been deleted in master branch ，so   I will close this pull request。